Realm/ObjectServerTests/RLMSyncSessionRefreshHandle+ObjectServerTests.h

```diff
@@ -0,0 +1,26 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2017 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "RLMSyncSessionRefreshHandle.h"
+
+@interface RLMSyncSessionRefreshHandle (ObjectServerTests)
+
++ (void)rlmTestUtils_calculateFireDateUsingTestLogic:(BOOL)forTest
```

Comment:
there's no need to prefix this method like this given that it's defined and implemented in our unit tests only. Could just be `+calculateFireDateUsingTestLogic:blockOnRefreshCompletion:`